server/haztrak/urls.py

```diff
@@ -22,8 +22,8 @@
 
 urlpatterns = [
     path("admin/", admin.site.urls),
-    path("api/trak/", include("apps.trak.urls")),
-    path("api/site/", include("apps.sites.urls")),
+    path("api/v1/trak/", include("apps.trak.urls")),
```

Comment:
We'll be unable to make use of this version information in the URL in our requests objects. 

see the DRF examples here https://www.django-rest-framework.org/api-guide/versioning/#namespaceversioning